src/main/resources/locales/en/case-reference-number-search-results.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Search result",
-    "subheader": "1 result successfully found",
+    "subheader": "1 found",
```

Comment:
I can't see the Welsh translation for this change. Guess we need to change Welsh too?

Reply:
Committed changes